### PR TITLE
Please add my third homework

### PR DESCRIPTION
**Pull request by AdrianFilimon1707**

### CODE REVIEW

File: Function_Test_Work.py [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    years = description[1]
+
+    data = {
+        country: [{
+            'year': year, 'coverage': coverage
+        }
+            for coverage, year in zip(coverages, years)
+        ]
+        for country, coverages in raw_data
+    }
```
Comment (mariuspodean) [inline Function_Test_Work.py]: You can increase code readability by formatting the code as follows:
```python
{
    country: [
        {
            'year': year,
            'coverage': coverage
        }
        for coverage, year in zip(coverages, years)
    ]
    for country, coverages in raw_data
}
```

File: Function_Test_Work.py [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+# >>> {'2019': [('Romania', 84), ('Germany', 95), ..., ('Latvia', 85)]}
+
+def year_data(dataset, year):
+    func2 = {
+        year: [(country, index['coverage'])
+               for country in dataset.keys()
+               for index in dataset[country]
+               if index['year'] == str(year)]
+    }
+    return func2
```
Comment (mariuspodean) [inline Function_Test_Work.py]: Same as above concerning formating

File: Function_Test_Work.py [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+
+    return data
+
+
+# get_year_data(dataset, "2019")
+#
+# >>> {'2019': [('Romania', 84), ('Germany', 95), ..., ('Latvia', 85)]}
+
+def year_data(dataset, year):
+    func2 = {
```
Comment (mariuspodean) [inline Function_Test_Work.py]: Why is this called **func2** if it's a dict?

File: Function_Test_Work.py [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+data_year = year_data(dataset, 2015)
+
+print(data_year)
+
+
+# {'Romania': [('2019', 84), ('2018', 86), ..., ('2011', 72)]}
+
+def get_country_data(dataset, country):
+    func3 = {country: [(value['year'], value['coverage'])
```
Comment (mariuspodean) [inline Function_Test_Work.py]: Formatting

**Comment by AdrianFilimon1707** [comment]
???

File: Function_Test_Work.py (hunk 3 above)
Comment (AdrianFilimon1707) [reply Function_Test_Work.py]: in my mind: because this function returns a dict practically

Comment (mariuspodean) [reply Function_Test_Work.py]: yes, but the variable contains a dict; please git it a meaningful name

**Pull request merged**